docs/tutorial/voting-tutoral.md

```diff
@@ -0,0 +1,214 @@
+From the `.json` file, you'll see the account _id, private-key and public-key. You'll need to copy the public key from the .json and link it to your account in the console.
+
+- Open Calimero [Console](https://app.calimero.network/dashboard)
+- Click on **Security**
+- Click on **Custodial Accounts**
+- Search for your custodian account ID in the list and click on the  `⋮` three dots menu.
+- Click on Add public key
+
+![](../../static/img/public_keys.png)
+
+## Create sub account
+
+Next, we will create sub account which will be used to deploy the account that we have previously built. Run the following command:
+
+ ```bash
+near create-account <SUB_ACCOUNT_ID>.SHARD_ID.calimero.testnet --masterAccount SHARD_ID.calimero.testnet --networkId SHARD_ID-calimero-testnet--nodeUrl https://api.calimero.network/api/v1/shards/SHARD_ID.calimero.testnet/neard-rpc/  
+ ```
+
+For our example, this will be
+
+ ```bash
+near create-account yt-voting-example-1.demos.calimero.testnet--masterAccount demos.calimero.testnet --networkId demos-calimero-testnet--nodeUrl https://api.calimero.network/api/v1/shards/demos.calimero.testnet/neard-rpc/  
+ ```
+
+- Replace **<SUB_ACCOUNT_ID>** with your sub account name which can be anything for e.g yt-voting-example
+- Replace **<SHARD_ID>** with your shard name
+
+After, the command has ben ran, you can check in the **Custodial Accounts** section in your console to view the sub account you created.
+
+![](../../static/img/yt-sub-account.png)
+
+## Deploy your NEAR contract
+
+The next step, is to deploy this contract in the private shard. In your cloned repository, navigate to `deploy.calimero.sh`. We will be using the second command which is `near deploy`
+
+```bash
+near deploy \
+  --accountId "" \
+  --initFunction new --initArgs '{"question": "Which blockchain is best?", "options": ["NEAR","Bitcoin"]}' \
+  --wasmFile target/wasm32-unknown-unknown/release/poll.wasm \
+  --nodeUrl "calimero-rpc-node-url" \
+  --networkId "$1-calimero-testnet"
+```
+
+- Replace the _accountID_ to your sub account name. In our example this will be  `yt-voting-example-1.demos.calimero.testnet`
+- Replace _networkID_ to your shard name. In our example this will be `demos-calimero-testnet`
+- For the `calimero-rpc-node-url`, go to the Calimero [Console dashboard](https://app.calimero.network/dashboard) and copy the the near RPC endpoint
+
+For our example, this will be
+
+```bash
+near deploy \
+  --accountId "yt-voting-example-1.demos.calimero.testne" \
+  --initFunction new --initArgs '{"question": "Which blockchain is best?", "options": ["NEAR","Bitcoin"]}' \
+  --wasmFile target/wasm32-unknown-unknown/release/poll.wasm \
+  --nodeUrl "https://api.calimero.network/api/v1/shards/demos-calimero-testnet/neard-rpc/  " \
+  --networkId "demos-calimero-testnet"
+```
+
+If the contract was successfully deployed you can check this on the [Explorer > Transactions](https://app.calimero.network/dashboard/explorer/transactions) on your the console.
```

Comment:
You can check the explorer regardless of the deployment success/failure. But that's just a phrasing issue